code/frameworks/angular/src/server/angular-cli-webpack-13.x.x.ts

```diff
@@ -54,7 +51,13 @@ exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }
    */
   const entry = [
     ...baseConfig.entry,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Here for 13.0.0. : https://github.com/angular/angular-cli/blob/13.0.x/packages/angular_devkit/build_angular/src/webpack/configs/styles.ts#L42-L67

or here for 15 : 
https://github.com/angular/angular-cli/blob/9ca921b90b65264ec2a1adc8addd345ec42a3ea3/packages/angular_devkit/build_angular/src/webpack/utils/helpers.ts#L137

angular create something like : 

```
entry : {
  styles : [styles.css]
  [custom bundle] : [other.css] 
}
```

so the code here try to flatten styles and add it with the one provided by sb `baseConfig`.

but this code take only styles and not others bundle 

so that why we still have this issue IMHO : 
https://github.com/storybookjs/storybook/issues/17711 (https://github.com/storybookjs/storybook/pull/17888)


i think here we need to have something like : 
`const entry = { ...cliConfig.entry, main: baseConfig.entry };`

Stroybook only provide array of entry so we can take in in man (or `storybookMain`) and keep all other Record from angular